src/python/pants/backend/project_info/rules/dependencies.py

```diff
@@ -20,10 +27,16 @@ def register_options(cls, register):
     super().register_options(register)
     register(
       '--transitive',
-      default=True,
+      default=False,
```

Comment:
The V1 task will change its default to `False` in 1.27.0.dev0. I changed this one now to make sure we don't forget to also update the V2 implementation.